src/composables/graph/useGraphNodeManager.ts

```diff
@@ -204,8 +204,13 @@ export const useGraphNodeManager = (graph: LGraph): GraphNodeManager => {
 
     return {
       id: String(node.id),
-      title: node.title || 'Untitled',
-      type: node.type || 'Unknown',
+      title: typeof node.title === 'string' ? node.title : '',
+      type:
+        node.type ||
+        node.constructor?.comfyClass ||
+        node.constructor?.title ||
+        node.constructor?.name ||
+        'Unknown',
```

Comment:
With this many options, would you mind pulling it out into a named const, for readability?